src/traces/scattergl/index.js

```diff
@@ -161,6 +165,7 @@ function sceneOptions(gd, subplot, trace, positions, x, y) {
     return opts;
 }
 
+
```

Comment:
Again here. Please don't commit lines like this.

Reply:
Ok, but in this case all functions are separated by 2 new lines and here by just one, so hope you don't mind a bit of perfectionism